manifests/config.pp

```diff
@@ -91,7 +95,7 @@
   file { 'consul config.json':
     ensure  => present,
     path    => "${consul::config_dir}/config.json",
-    content => consul_sorted_json($config_hash),
+    content => consul_sorted_json($config_hash, $pretty),
```

Comment:
We should do this change for `service`, `check` and `watch` as well for consistency.